packages/core/src/domain/telemetry/telemetry.spec.ts

```diff
@@ -81,6 +81,19 @@ describe('telemetry', () => {
     expect(notifySpy.calls.mostRecent().args[0].experimental_features).toEqual(['foo'])
   })
 
+  it('should contains runtime env', () => {
+    addExperimentalFeatures(['foo' as ExperimentalFeature])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏you probably don't need the flag for this test
```suggestion
  it('should contains runtime env', () => {
```

Reply:
Right 🤦‍♂️